go/arrow/memory/checked_allocator.go

```diff
@@ -160,14 +160,24 @@ func (a *CheckedAllocator) AssertSize(t TestingT, sz int) {
 				break
 			}
 			callersMsg.WriteString("\t")
-			callersMsg.WriteString(frame.Function)
-			callersMsg.WriteString(fmt.Sprintf(" line %d", frame.Line))
+			if fn := frame.Func; fn != nil {
+				callersMsg.WriteString(fmt.Sprintf("%s+%x", fn.Name(), frame.PC-fn.Entry()))
```

Comment:
offset in bytes from func start, if we have fn (we don't for inlined funcs

Reply:
add this as a comment here so that we don't lose the information? This is good to know so we don't look at this in the future and go "wtf is this doing!"

Reply:
Done in c95516aaef66bf4de3d925fde030eddbc5a78af7